### PR TITLE
New Dropbox Cheat Sheet

### PR DESCRIPTION
**Pull request by DPAUTHe**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

-  New Instant Answer
    -  Cheat Sheets: **`New Dropbox Cheat Sheet`**

---

Instant Answer Page: https://ddgc-staging.duckduckgo.com/ia/view/dropbox_cheatsheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@DPAUTHe Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by DPAUTHe** [comment]
@moollaza When will my cheat sheet be added on beta?

**Comment by VeerpalBrar** [comment]
@DPAUTHe  If you created this using codio.com then you can test the cheat sheet using duckpan server. 
Information can be found in the duck duck hacks documentation, found [here](http://docs.duckduckhack.com/walkthroughs/programming-syntax.html): Just scroll down to "Interactively Test Your Cheat Sheet". 

If you have not been using codio, [this](http://docs.duckduckhack.com/welcome/setup-dev-environment.html) page will walk you through it.

File: share/goodie/cheat_sheets/json/dropbox.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                "key": "Esc",
+                "val": "Deselect these files (and hide these instructions)"
+            },
+            {
+                "key": "/",
+                "val": "Search"                              
+            },
+            {
+                "key": "Enter",
+                "val": "Deselect these files (and hide these instructions)"
```
Comment (VeerpalBrar) [inline share/goodie/cheat_sheets/json/dropbox.json]: I think you can remove the word "these". So 'Deselect files (and hide instructions)'

**Comment by DPAUTHe** [comment]
I used codio to build it. My question is when will it be added to beta.duckduckgo.com ?

**Pull request closed without merging**